src/main/java/seedu/foodrem/commons/core/Version.java

```diff
@@ -62,9 +47,25 @@ public static Version fromString(String versionString) throws IllegalArgumentExc
         }
 
         return new Version(Integer.parseInt(versionMatcher.group(1)),
-                Integer.parseInt(versionMatcher.group(2)),
-                Integer.parseInt(versionMatcher.group(3)),
-                versionMatcher.group(4) == null ? false : true);
+            Integer.parseInt(versionMatcher.group(2)),
+            Integer.parseInt(versionMatcher.group(3)),
+            versionMatcher.group(4) != null);
```

Comment:
This issue is still present, that after line breaks there's an indentation of 4 spaces instead of 8 as required by cs2103t coding style.